From 3ad00e25b9425ff0fa4c44029d07b46ca2f20540 Mon Sep 17 00:00:00 2001
From: "Mohamad Azman, Syaza Athirah" <syaza.athirah.mohamad.azman@intel.com>
Date: Thu, 16 Jul 2020 18:00:51 -0400
Subject: [PATCH 1/2] libbpf: add txtime field in xdp_desc struct

Add a xdp_desc entry to specify packet LaunchTime for drivers
that support the feature.

Signed-off-by: Mohamad Azman, Syaza Athirah <syaza.athirah.mohamad.azman@intel.com>
---
 include/uapi/linux/if_xdp.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/uapi/linux/if_xdp.h b/include/uapi/linux/if_xdp.h
index a78a809..7c5168c 100644
--- a/include/uapi/linux/if_xdp.h
+++ b/include/uapi/linux/if_xdp.h
@@ -104,6 +104,7 @@ struct xdp_desc {
 	__u64 addr;
 	__u32 len;
 	__u32 options;
+	__u64 txtime;
 };

 /* UMEM descriptor is __u64 */
-- 
2.7.4

